library/net/ftp/set_socket_spec.rb

```diff
@@ -4,5 +4,5 @@
 describe "Net::FTP#set_socket" do
   # TODO: I won't spec this method, as it is not used
   #       anywhere and it should be private anyway.
-  #it "needs to be reviewed for spec completeness"
+  it "needs to be reviewed for spec completeness"
 end
```

Comment:
I would just remove the file entirely.
The comment seems wrong, that method seems public in MRI 2.6 at least.

Reply:
Ah it's fine, the comment says `it should be private`, not that it is already private.